docs/src/content/docs/fr/guides/components.mdx

```diff
@@ -205,3 +205,49 @@ Le code ci-dessus génère ce qui suit sur la page :
 Une liste de toutes les icônes disponibles est affichée ci-dessous avec leurs noms associés. Cliquez sur une icône pour copier le code du composant.
 
 <IconsList />
+
+### Code
+
+Utilisez le composant `<Code>` pour rendre le code en évidence syntaxique lorsque l'utilisation d'un [bloc de code Markdown](/fr/guides/authoring-content/#blocs-de-code) n'est pas possible, par exemple, pour rendre des données provenant de sources externes comme des fichiers, des bases de données ou des API.
```

Comment:
```suggestion
Utilisez le composant `<Code>` pour afficher du code avec coloration syntaxique lorsque l'utilisation d'un [bloc de code Markdown](/fr/guides/authoring-content/#blocs-de-code) n'est pas possible, par exemple, pour afficher des données provenant de sources externes comme des fichiers, des bases de données ou des API.
```

- I personally have never heard of "évidence syntaxique" before and I cannot find any reference to it. We have also been [using](https://starlight.astro.build/fr/guides/authoring-content/#blocs-de-code) "coloration syntaxique" in the past so I would suggest to use that instead.
- When translating the [overrides references](https://starlight.astro.build/reference/overrides/) page which contains a lot of time the verb "render", we decided that "rendre" does not really fit the context and is even used a lot for that matter. We decided to use "afficher" instead when possible.

Reply:
"évidence syntaxique" 
I thought it was weird, but I messed it up and couldn't find the right words at the time, so I'm glad you noticed! 🙏🏼